addons/osfstorage/models.py

```diff
@@ -304,7 +304,9 @@ def create_version(self, creator, location, metadata=None):
         if metadata:
             version.update_metadata(metadata, save=False)
 
-        version.region = self.target.osfstorage_region
+        if 'osfstorage_region' in self.target.__dict__:  # clear cache if cached
```

Comment:
Did we ever sort out which tests were failing and if the cache should be cleared in those instead?

Reply:
Here's the test log https://gist.github.com/Johnetordoff/e80d4cd97ab246577447bc24c5dcdfcc other then `test_validate_social_profile_website_many_different` all these test fails are plausibly related to cached property problems. In this PR: https://github.com/CenterForOpenScience/osf.io/pull/10048/files#diff-29ba2031d64258cef5a604953b099885bf5f786f7efdf2df3debd05ba55d5e14R761 you can see the places I fixed these test by [checking for attributes](https://github.com/CenterForOpenScience/osf.io/pull/10048/files#diff-966024d7414683bec3c3de333eabee6f7f35586781f64bd0324b6e0894dc4135R114-R116) and [reloading](https://github.com/CenterForOpenScience/osf.io/pull/10054/files#diff-2970b019eaf6b19803b82aae2e2e7e0a0f5c95bcdfda2a6ef1f2c070bf989fcbL982) them. But I'm not 100% sure if those are related.

Reply:
Ah, I see. For now let's postpone this one until after staging3 deployment and QA.